web/app/themes/imb/header.php

```diff
@@ -4,6 +4,18 @@
 
 <?php do_action('after_body_open_tag'); ?>
```

Comment:
Do we still need this action? after_body_open_tag

Reply:
Good point! Now removed :D